### PR TITLE
commit 1 in atef local branch

### PR DESCRIPTION
**Pull request by MoatasemEbrahim**

### CODE REVIEW

File: src/components/Layout/Layout.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ const Layout = () => (
         Content
       </Container>
     </main>
+    safihewiugfiugfewyuewgfgoiewugfo
```
Comment (mohamedatef73) [inline src/components/Layout/Layout.tsx:14]: bad format

**Review by mohamedatef73 — COMMENTED** [review]
ay 7aga

File: src/components/atefComponent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const index = () => {
```
Comment (mohamedatef73) [inline src/components/atefComponent/index.tsx:3]: u have to change this

**Review by mohamedatef73 — CHANGES_REQUESTED** [review]

File: src/components/atefComponent/index.tsx (hunk 2 above)
Comment (MoatasemEbrahim) [reply src/components/atefComponent/index.tsx:3]: sldfsldhflds

**Review by mohamedatef73 — APPROVED** [review]
done

**Pull request closed without merging**